File: C#-CarRacingProject/Domain/Cursa.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace ProiectCSharpVersion.Domain
{
    class Cursa:Entity<int>
    {
        public int CapCilindrica { get; set; }
        public Cursa(int capCilindrica)
        {
            this.CapCilindrica = capCilindrica;
        }

        public override string ToString()
        {
            return "Cursa: capCilindrica=" + CapCilindrica;
        }
    }
}

File: CarRacing_Client_Server_HibernatePersistence/CurseModel/src/main/java/curse/model/Cursa.java
package curse.model;

public class Cursa extends Entity<Integer>{
    private Integer capCilindrica;
    public Cursa(){

    }
    public Cursa(Integer capCilindrica){
        this.capCilindrica=capCilindrica;
    }
    public Integer getCapCilindrica() {
        return capCilindrica;
    }

    public void setCapCilindrica(Integer capCilindrica) {
        this.capCilindrica = capCilindrica;
    }

    @Override
    public String toString() {
        return "Cursa{"+
                "capCilindrica="+capCilindrica+
                "mc}";
    }
}
